### ISSUE TITLE
invalid css name generated

### ISSUE THREAD

**Issue by semtselishev** [DanielAmenou/rollup-plugin-lib-style#7]
Hi Daniel, thanks for the great plugin. 

After quite some time, I was able to set it up as I want, but I came across a few problems down the road, so I want to report them here.
It's for  following setup and 2.3.0 version
```
      libStylePlugin({
        loaders: [
          {
            name: 'sass',
            regex: /\.(sass|scss)$/,
            // loadPaths defines the global imports in SCSS ( to support pretty imports for Mixins)
            process: ({ filePath }) => ({ code: sass.compile(filePath, { loadPaths: ['./'] }).css.toString() }),
          },
        ],
        scopedName: '[hash:base64]',
        customCSSInjectedPath: id => '/' + id.split('/').pop(),
        customCSSPath: id => id.replace(process.cwd() + '/src', '').replace(/\\/g, '/'),
        postCssPlugins: [postcssRTLCSS({ mode: Mode.override })],
      }),
```
`[hash:base64]` can produce an invalid classname started with a number like `.7f949f` or containing a slash inside like `f5Sf/hJ`. https://webpack.js.org/loaders/css-loader/ for examples changes all special symbols to avoid such problems 

> Note: all reserved (<>:"/\|?*) and control filesystem characters (excluding characters in the [local] placeholder) will be converted to -.

**Comment by semtselishev** [comment]
Also, noticed another issue. All classes in one scss/css file always get the same `hash` when  `[hash:base64]` is passed , though it should be a unique one for each class

**Comment by DanielAmenou** [comment]
Hi @semtselishev, thank you for your feedback!

I’ll do my best to fix these issues soon. Since you’ve raised a few distinct points, could you please open separate issues for each one? This will help me track and resolve them more effectively.

Thanks again for your input and for helping improve the plugin!

**Comment by semtselishev** [comment]
ok, I created a few tickets:
1) related to SASS https://github.com/DanielAmenou/rollup-plugin-lib-style/issues/8
2) CSS imports issue https://github.com/DanielAmenou/rollup-plugin-lib-style/issues/9

and I cleaned this ticket to have here only one problem

**Comment by DanielAmenou** [comment]
Hi @semtselishev ,

Thank you for bringing up this issue. I’ve released a new version, rollup-plugin-lib-style@2.3.1, which addresses this issue.
